format/index.md

```diff
@@ -785,6 +794,15 @@ include additional [fields] in the response.
 GET /articles?include=author&fields[articles]=title,body&fields[people]=name
 ```
 
+If both query parameters `include` and `fields[TYPE]` are provided by the client, 
+values of `TYPE` **MUST** either refer to the primary data or be listed in 
+`include`, otherwise the server **MUST** return `400 Bad Request`.
+
+If both query parameters `include` and `fields[TYPE]` are provided by the client,
+and a given related object is requested in `include` but reference to it is not 
+requested in the corresponding `fields[TYPE]`, the server **MUST** return 
+`400 Bad Request`.
```

Comment:
The logic from the above comment applies here too. This is the enforcement of the second clause of the PR. Reworded it to make it more clear, although it looks bulky now.

>  It seems to suggest that `fields` must be present for each `include`. Why?

Not sure I understand. It says "If both query parameters `include` and `fields[TYPE]` are provided by the client". It does not suggest that they both "must be present", it only considers what should the server do when client decided to provide both of them.